indent/javascript.vim

```diff
@@ -458,7 +459,7 @@ function GetJavascriptIndent()
   if l:line =~ '^[])}]\|^|}'
     return max([indent(num),0])
   elseif num
-    return indent(num) + s:W + switch_offset + bL + isOp
+    return max([indent(num) + get(l:,'case_offset',s:W) + switch_offset + bL + isOp, 0])
```

Comment:
I think case_offset visibility should be change as s`witch_offset`. Just set it line 431 to` s:sw()`.
Or better way, set it to `0` line 431 and line 440 to  `cino(`=`) -s:W` . So line 462  should be
`return max([indent(num) + s:W + switch_offset + +case_offset + bL + isOp, 0])`